app/views/events/_form.html.erb

```diff
@@ -21,6 +21,11 @@
     <%= form.text_area :description, id: :event_description %>
   </div>
 
+  <div class="field">
+    <%= form.label :speaker %>
+    <%= form.text_area :speaker, id: :event_speaker %>
```

Comment:
text_area is used for large blocks of text, text_field would be better in cases like this.